### PR TITLE
Standardise company success messages for add and delete company and u…

### PR DESCRIPTION
**Pull request by Green-Tea-123**
…pdate sample address book.
partial fix for #157 .
Also updating the sample address book so that no test items are inside. The user guide will probably need to be updated with the new sample address book.

### CODE REVIEW

**Review by wassilililily — APPROVED** [review]
Simple changes LGTM

File: src/main/java/seedu/address/logic/commands/AddCompanyCommand.java
Comment (Quasant) [inline src/main/java/seedu/address/logic/commands/AddCompanyCommand.java:32]: Just to confirm, so the desired format is as follows?

image

I'm okay with it, but don't merge this PR yet till Alek takes a look as it relates to him.

Comment (Quasant) [reply src/main/java/seedu/address/logic/commands/AddCompanyCommand.java:32]: However, do note that on the default window size, text for even yours would be partially covered.

image

**Review by Quasant — APPROVED** [review]
Yea I'm agreeable to it. My concern would be that displaying everything out for contacts and jobs would result in text overflowing. Gonna pull Alek in as well since this pertains to him.

File: src/main/java/seedu/address/logic/commands/AddCompanyCommand.java
Comment (Green-Tea-123) [reply src/main/java/seedu/address/logic/commands/AddCompanyCommand.java:32]: I'm not sure how bad it is to overflow like that since the scrollbar is present. The invalid command format also causes text to be displayed out of the visible box.

**Review by AlekKwek18 — APPROVED** [review]
Thank you for tagging me to this PR. LGTM if this is the desired format. I will follow this

**Pull request merged**
